symengine/derivative.cpp

```diff
@@ -670,6 +641,46 @@ class DiffImplementation
             }
         }
     }
+
+    static RCP<const Basic> diff(const LeviCivita &self,
+                                 const RCP<const Symbol> &x)
+    {
+        return diff0(self, x);
+    }
+
+    template <typename T>
+    static RCP<const Basic> diff0(const T &self, const RCP<const Symbol> &x)
```

Comment:
How about changing this to `static RCP<const Basic> diff(const Function &self, const RCP<const Symbol> &x)` ?